PostBoard.Api/Controllers/BirthdayController.cs

```diff
@@ -69,32 +69,30 @@ public  async Task<IActionResult>  UpdateAsync([FromRoute] int id, [FromBody] Bi
             return BadRequest("Validation error.");
         }
 
-        var birthday = await _dbcontext.Birthdays.FirstOrDefaultAsync(b => b.Id == id);
-        if (birthday == null)
+       
+        try
+        {
+            await _birthdayService.UpdateAsync(id, input);
+            return NoContent();
+        }
+        catch (InvalidOperationException)
         {
             return NotFound($"Birthday with Id={id} not found.");
         }
-        
-        birthday.UserFullName = input.UserFullName;
-        birthday.Date = input.Date;
-       await _dbcontext.SaveChangesAsync();
-        
-        return Ok(birthday);
```

Comment:
In Update method of Controller you have to:
1) update entity using Service.Update method;
2) get entity using Service.GetById method;
3) return entity using `Ok(entity)` method.